### ISSUE TITLE
Low serum alanine aminotransferase

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5946]
**Preferred term label:** Low serum alanine aminotransferase

**Synonyms** Low serum ALT; Alanine Aminotransferase decreased

**Definition (free text, please give PubMed ID)** An abnormally low concentration in the circulation of alanine aminotransferase (ALT), which is an enzyme that catalyzes the transfer of amino groups to form the hepatic metabolite oxaloacetate., PMID:25013373 

**Parent term (use hpo.jax.org/app)** Elevated hepatic transaminase HP:0002910

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
There is no medically relevant "low" ALT level -- basically, the test is whether ALT is increased or not.
